packages/core/src/env-set/oidc.ts

```diff
@@ -1,66 +1,88 @@
 import crypto, { generateKeyPairSync } from 'crypto';
-import { readFileSync, writeFileSync } from 'fs';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
 
 import { getEnv } from '@silverhand/essentials';
 import inquirer from 'inquirer';
 import { nanoid } from 'nanoid';
 
+import { exportJWK } from '@/utils/jwks';
+
 import { appendDotEnv } from './dot-env';
 import { allYes, noInquiry } from './parameters';
+import { getStringArrayEnv } from './utils';
 
 /**
- * Try to read private key with the following order:
+ * Try to read private keys with the following order:
  *
- * 1. From `process.env.OIDC_PRIVATE_KEY`.
- * 2. Fetch path from `process.env.OIDC_PRIVATE_KEY_PATH` then read from that path.
+ * 1. From `process.env.OIDC_PRIVATE_KEYS`.
+ * 2. Fetch path from `process.env.OIDC_PRIVATE_KEY_PATHS` then read from that path.
  *
  * If none of above succeed, then inquire user to generate a new key if no `--no-inquiry` presents in argv.
  *
- * @returns The private key for OIDC provider.
+ * @returns The private keys for OIDC provider.
  * @throws An error when failed to read a private key.
  */
-const readPrivateKey = async (): Promise<string> => {
-  const privateKey = getEnv('OIDC_PRIVATE_KEY');
+const readPrivateKeys = async (): Promise<string[]> => {
+  const privateKeys = getStringArrayEnv('OIDC_PRIVATE_KEYS');
 
-  if (privateKey) {
-    return privateKey;
+  if (privateKeys.length > 0) {
+    return privateKeys;
   }
 
-  const privateKeyPath = getEnv('OIDC_PRIVATE_KEY_PATH', './oidc-private-key.pem');
+  const privateKeyPaths = getStringArrayEnv('OIDC_PRIVATE_KEY_PATHS', ['./oidc-private-key.pem']);
```

Comment:
should we keep a private key folder to store these rotatable keys?  Otherwise, it has to be spread out under the root path right.

Reply:
We can leave this to the user.
```
OIDC_PRIVATE_KEY_PATHS=["./folder/file1.pem", "./folder/file2.pem"]
```